grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx

```diff
@@ -382,15 +382,25 @@ export const IntegrationForm = observer(
 
     async function onWebhookTestClick(): Promise<void> {}
 
-    async function onFormSubmit(formData): Promise<void> {
+    async function onFormSubmit(formData: FormFields): Promise<void> {
       const labels = labelsRef.current?.getValue();
-      const data = { ...formData, labels };
+
+      const data: OmitReadonlyMembers<ApiSchemas['AlertReceiveChannel']> = {
+        labels,
+        ...formData,
+      };
+
+      if (formData.integration !== 'servicenow') {
+        delete data.additional_settings;
+      }
+
       const isCreate = id === 'new';
 
       try {
         if (isCreate) {
           await createNewIntegration();
         } else {
+          // @ts-ignore
```

Comment:
Can we pls remove it and to get rid of the error do:
![image](https://github.com/grafana/oncall/assets/12073649/4afc763d-372f-4981-b39a-04aabaf791a9)

Reply:
Oops, forgotten, wasn't on purpose